tensorflow/python/ops/nn_grad.py

```diff
@@ -421,6 +421,13 @@ def _MaxPoolGrad(op, grad):
                                    data_format=op.get_attr("data_format"))
 
 
+@ops.RegisterGradient("MaxPoolGrad")
```

Comment:
This needs a test, perhaps using tf.test.compute_gradient_error, to prove that this is right :)